azext_edge/edge/providers/orchestration/resources/connector/opcua/certs.py

```diff
@@ -221,9 +217,9 @@ def client_add(
         resource_group: str,
         public_key_file: str,
         private_key_file: str,
-        subject_name: str,
-        application_uri: str,
         overwrite_secret: bool = False,
+        subject_name: Optional[str] = None,
+        application_uri: Optional[str] = None,
         public_key_secret_name: Optional[str] = None,
         private_key_secret_name: Optional[str] = None,
     ) -> dict:
```

Comment:
The warning as written on line 227 is not conveying (as well) the same point written in the comment on line 226, recommend some wording change.